src/app/components/SwapWidget/SwapStepOne.jsx

```diff
@@ -55,7 +55,7 @@ const SwapStepOne = ({ isPopUpOpen, handlePopUp, supportedAssets, handleSendAmou
             onSubmit={handleSwapSubmit} 
             receiveSymbol={receiveSymbol} 
             sendSymbol={sendSymbol}
-            initialValues={{ sendSymbol, receiveSymbol, receiveAddress: '', refundAddress: '' }}
+            initialValues={{ sendSymbol, receiveSymbol, receiveAddress: undefined, refundAddress: undefined }}
```

Comment:
ok so i realized adding an initalValue for receiveAddress helps with the error styling of the dropdown (because dirty prop is now true after it doesnt == undefined anymore) buttt when you switch assets it resets the receiveAddress field to undefined. trying to think of a way to fix this.